libraries/eosiolib/capi/eosio/domain.h

```diff
@@ -19,41 +19,41 @@ extern "C" {
 
 
 /**
- *  Verifies that @ref domain is an existing domain name.
+ *  This method verifies that @a domain is an existing domain name.
  *  @param domain - domain name to check, a null terminated string
  */
 __attribute__((eosio_wasm_import))
 int32_t is_domain(const char* domain);
 
 /**
- *  Verifies that @ref username is an existing username in given scope.
```

Comment:
I like the old version more. The sentence is simpler. when one read the documentation, first look at the name of the method, then at the description and get a sentence: _`is_username` verifies that username is an existing username in given scope._

Reply:
returned previous piece of text